tfprotov5/internal/toproto/list_resource.go

```diff
@@ -18,6 +18,15 @@ func GetMetadata_ListResourceMetadata(in *tfprotov5.ListResourceMetadata) *tfplu
 	}
 }
 
+func ListResource_ListResourceEvent(in *tfprotov5.ListResourceResult) *tfplugin5.ListResource_Event {
+	return &tfplugin5.ListResource_Event{
+		DisplayName:    in.DisplayName,
+		ResourceObject: DynamicValue(in.Resource),
+		Identity:       ResourceIdentityData(in.Identity),
+		Diagnostic:     Diagnostics(in.Diagnostics),
```

Comment:
Can we fix `Diagnostic` to be plural for consistency reasons? I think that's what we use everywhere

Reply:
Has to be fixed in the protocol (I think core has it "wrong" [too](https://github.com/hashicorp/terraform/blob/2c721982099c560a353fd3d9ddd87c3f60e93f7e/docs/plugin-protocol/tfplugin5.proto#L847))